SignalServiceKit/src/Devices/OWSDevice.m

```diff
@@ -52,7 +52,7 @@ - (BOOL)mayHaveLinkedDevices:(YapDatabaseConnection *)dbConnection
         if (!self.mayHaveLinkedDevicesCached) {
             self.mayHaveLinkedDevicesCached = @([dbConnection boolForKey:kTSStorageManager_MayHaveLinkedDevices
                                                             inCollection:kTSStorageManager_OWSDeviceCollection
-                                                            defaultValue:YES]);
+                                                            defaultValue:NO]);
```

Comment:
We want to send a sync message in one of two cases:

* We know that we have linked devices by inspecting `SignalRecipient.devices` for our own instance of `SignalRecipient`.
* There might be a new linked device that we've just linked.  This flag is intended to reflect the latter.